tests/unit/fee-services.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { describe, expect, vi } from 'vitest'
+import axios from 'axios'
+import FeeServices from '../../src/services/fee-services'
+
+vi.mock('axios', () => ({
+  get: vi.fn(),
+  all: vi.fn(),
+  spread: vi.fn()
+}))
+
+vi.mock('axios')
```

Comment:
What does this do (compared with line 5)?

Reply:
Thanks for spotting that. With this default mock (`vi.mock'), code on line 5 is not needed. I will remove it.

Reply:
Removed